README.md

```diff
@@ -41,6 +41,11 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
+for(var i = 0; i<foods.length; i++){
+
```

Comment:
close, what should we do inside this loop? in order to print out all the names of the fruits inside the first item in the `foods` array, we need to write some code like:

```js
for(var i = 0; i < foods[0].length; i++){
  console.log("I want to eat a " + foods[0][i]);
}
```